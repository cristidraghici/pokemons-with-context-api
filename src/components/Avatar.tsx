import { useGlobalContext } from '../hooks'

const Avatar = (): JSX.Element => {
  const { state } = useGlobalContext()

  return <>{state.pokemonAvatar && <img src={state.pokemonAvatar} alt={state.pokemonName} />}</>
}

export default Avatar
