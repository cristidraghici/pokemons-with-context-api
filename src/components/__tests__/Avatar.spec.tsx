import { render } from '@testing-library/react'
import Avatar from '../Avatar'

describe('./components/Avatar', () => {
  it('matches snapshot', () => {
    const { container } = render(<Avatar />)
    expect(container).toMatchSnapshot()
  })
})
