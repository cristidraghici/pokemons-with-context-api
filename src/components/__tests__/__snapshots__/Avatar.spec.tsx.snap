// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./components/Avatar matches snapshot 1`] = `
<div>
  
</div>
`;
